internal/gensupport/send_test.go

```diff
@@ -6,6 +6,7 @@ package gensupport
 
 import (
 	"context"
+	"errors"
```

Comment:
This module still supports 1.11 today. Could you please change this to xerrors.

Reply:
changed!